src/fides/api/ctl/deps.py

```diff
@@ -35,6 +35,12 @@ def get_db() -> Generator:
         db.close()
 
 
+def get_api_session() -> Session:
+    """Gets the shared database session to use for API functionality"""
+    db = sync_session()
```

Comment:
Both ops and ctl have a similar construct, you're using the ctl one here, but I have a PR that uses the ops one for the health check endpoints because the ctl one creates a new engine each time which seems to open up multiple connection pools - https://github.com/ethyca/fides/pull/1182.  I also might recommend using the ops session instead of this `sync_session` and later we can look into consolidating them between ops and ctl.

Reply:
whoops - missed that we still have the ops `deps.py` hanging around, definitely makes sense to use that - thanks!

i think i'll still need to update that `def _get_session() -> Session:` function to be importable and renamed, like we did [here](https://github.com/ethyca/fidesops/pull/1307/commits/b8038ef7ce7525a1d1eddb15ff7d23c48b5b7480#diff-c8f03de0dd032d797b343453252c118e478979d8ee9da1cfb686fb0f8c55bab1L41-L42) in the fidesops PR. does that sound OK to you?

Reply:
yes that makes sense